app/riot/src/client/preview/render-riot.test.ts

```diff
@@ -23,18 +24,21 @@ beforeEach(() => {
 
 describe('render a riot element', () => {
   it('should not work with nothing', () => {
+    // @ts-ignore
```

Comment:
Same here... `render`  should only have 1 argument but in every test, there is a magic `context` given too.